Formula/yelp-tools.rb

```diff
@@ -12,20 +12,45 @@ class YelpTools < Formula
     sha256 "66721baa2d6c6978cceac29f35e8112e96804b6ae0197d89175867f17caca2cd" => :mavericks
   end
 
+  depends_on "gettext" => :build
+  depends_on "gtk+3" => :run
+  depends_on "intltool" => :build
   depends_on "itstool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "libxslt" => :build
   depends_on "libxml2" => :build
-  depends_on "yelp-xsl"
+  depends_on "libxslt" => :build
+  depends_on "pkg-config" => :build
+
+  resource "yelp-xsl" do
+    url "https://download.gnome.org/sources/yelp-xsl/3.18/yelp-xsl-3.18.0.tar.xz"
+    sha256 "893620857b72b3b43ee3b462281240b7ca4d80292f469552827f0597bf60d2b2"
+  end
 
   def install
+    resource("yelp-xsl").stage do
+      system "./configure", "--disable-debug",
+                            "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}"
+      system "make", "install"
+      ENV.append_path "PKG_CONFIG_PATH", "#{prefix}/share/pkgconfig"
+    end
+
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
   end
 
+  def post_install
+    system "#{Formula["gtk+3"].opt_bin}/gtk3-update-icon-cache", "-f", "-t", "#{HOMEBREW_PREFIX}/share/icons/hicolor"
+  end
+
   test do
     system "#{bin}/yelp-new", "task", "ducksinarow"
     system "#{bin}/yelp-build", "html", "ducksinarow.page"
     system "#{bin}/yelp-check", "validate", "ducksinarow.page"
+
+    assert (pkgshare/"xslt/mallard/html/mal2html-links.xsl").exist?
+    assert (pkgshare/"js/jquery.syntax.brush.smalltalk.js").exist?
+    assert (pkgshare/"icons/hicolor/24x24/status/yelp-note-warning.png").exist?
+    assert (share/"pkgconfig/yelp-xsl.pc").exist?
```

Comment:
Add a second argument to these for the assertion message so we have a better output if they fail. Could consider looping through these filenames and make `"#{filename} doesn't exist"` as the assertion message